src/server/system_services/schemas/replication_configuration_schema.js

```diff
@@ -39,6 +39,29 @@ module.exports = {
                         }
                     }
                 }
+            },
+        },
+        aws_log_replication_info: {
+            type: 'object',
+            required: ['logs_location'],
+            properties: {
+                logs_location: {
+                    type: 'object',
+                    required: ['logs_bucket'],
+                    properties: {
+                        logs_bucket: {
+                            objectid: true // bucket id
```

Comment:
didn't we decide to access the logs will through the S3 bucket directly? 
of course, the user will have to provide a bucket that is accessible by the same account

Reply:
if remember correctly we decided that the logs bucket will also be a namespace bucket in Noobaa